contract/src/main.rs

```diff
@@ -1279,6 +1336,18 @@ pub extern "C" fn migrate() {
             );
         }
     }
+
+    let metadata_kind: NFTMetadataKind = utils::get_stored_value_with_user_errors(
+        NFT_METADATA_KIND,
+        NFTCoreError::MissingNFTMetadataKind,
+        NFTCoreError::InvalidNFTMetadataKind,
+    );
+    let mut nft_metadata_kind_list: BTreeMap<NFTMetadataKind, Requirement> = BTreeMap::new();
+    nft_metadata_kind_list.insert(metadata_kind, Requirement::Required);
+    runtime::put_key(
```

Comment:
Same here, we should extend the named keys, instead of modifying the existing one